DOCS: Adding instructions for linux Arch when using Make (#216)
larq/compute-engine: docs/build_arm.md

argument.

To natively build the LCE library and C++ example programs, first you need to
install the compiler toolchain on your target device. On Debian based systems like a
Raspberry Pi board with Raspbian, run the following command:
```
sudo apt-get install build-essential
```

> Or on an Arch based system like a Raspberry Pi board with Manjaro, run the following command instead:

Reviewer: ```suggestion
On an Arch based system like a Raspberry Pi board with Manjaro operating system, run the following command instead:
```

sudo apt-get update
sudo apt-get install crossbuild-essential-armhf
```

> For Arch based systems, the package is called `arm-linux-gnueabihf`. Run the following commands:

Reviewer: ```suggestion
On an Arch based systems, the package is called `arm-linux-gnueabihf`:
```
